docs/UserGuide.md

```diff
@@ -421,9 +450,9 @@ Deletes the specified person from the address book.
 <box type="definition" icon=":fa-solid-book:" light>
 
 <md>**Examples:**</md>
-* `list` followed by `delete n/Betsy` deletes the person with the name `Betsy`.
-* `find Betsy` followed by `delete Alex` will not delete the person named `Alex`.
-* `delete n/Betsy` deletes the person named `Betsy` if it is shown on the filtered list.
+* `list` followed by `rm n/Betsy` deletes the person with the name `Betsy`. It uses the shortcut command and parameter.
+* `find Betsy` followed by `delete n/Alex` will not delete the person named `Alex` as `Alex` does not appear in the contact list.
```

Comment:
Maybe instead of "contact" it might be more appropriate to use "filtered" as used in line 455.